### PR TITLE
feat

### PR DESCRIPTION
**Pull request by ckahub**
created productpage
chrome-capture-2022-2-26
e

### CODE REVIEW

**Comment by manishsom20** [comment]
No changes are required. I like your code structure, making of utility classes and proper naming convention

File: style/productpage.css [hunk 1]
```diff
@@ -0,0 +1,168 @@
+
+:root{
+    --primary-color:black;
+    --secondary-color:tan;
+    --active-color:aqua;
+}
+body{
+    background-color: tan;
+    padding: 0%;
+    border: 0%;
```
Comment (dusariudaykumar) [inline style/productpage.css:10]: instead of applying rest CSS on body, you can add it to a universal selector

File: pages/productpage.html [hunk 2]
```diff
@@ -1,15 +1,288 @@
+
 <!DOCTYPE html>
 <html lang="en">
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="author" content="chandrakant">
```
Comment (khushijohri2001) [inline pages/productpage.html:7]: You can add meta tags for Description too. Just a suggestion

**Review by khushijohri2001 — APPROVED** [review]
Code looks clean

File: style/productpage.css (hunk 1 above)
Comment (ckahub) [reply style/productpage.css:10]: ok in root

**Pull request merged**